### PR TITLE
[mdns] remove redundant ERR log

### PR DESCRIPTION
**Pull request by simonlingoogle**

### CODE REVIEW

File: src/mdns/mdns_avahi.cpp [hunk 1]
```diff
@@ -393,7 +393,6 @@ void PublisherAvahi::HandleGroupState(AvahiEntryGroup *aGroup, AvahiEntryGroupSt
 
     case AVAHI_ENTRY_GROUP_UNCOMMITED:
     case AVAHI_ENTRY_GROUP_REGISTERING:
-        otbrLogErr("Group ready.");
```
Comment (superwhd) [inline src/mdns/mdns_avahi.cpp:396]: Why deleting it instead of using a lower log level?

**Review by superwhd — APPROVED** [review]
LGTM with a question

File: src/mdns/mdns_avahi.cpp (hunk 1 above)
Comment (simonlingoogle) [reply src/mdns/mdns_avahi.cpp:396]: There are significant number of this logs. I guess the real question is "Is the log useful in anyway?".

Comment (superwhd) [reply src/mdns/mdns_avahi.cpp:396]: Then I think it's fine to remove it.

**Review by jwhui — APPROVED** [review]

**Pull request merged**